docs/manual/manual_english.md

```diff
@@ -0,0 +1,246 @@
+- In the right corner you can see the number of satellites of which the GPS module receives signal. The minimum required to obtain a valid location is **4 satellites**, although in this case more is better.
+
+### Odometer
+
+The screen you will most likely see most of the time is the odometer.
+In it you can see:
+- **Total distance** in the upper left: Counts the distance traveled from the last total distance reset (from the configuration menu). This distance **does NOT** vary when landing. It can be considered as the total and real distance that you have traveled during a complete stage.
+- **Partial distance** in the center: Counts the distance traveled since the last reset of partial distance.
+- **Heading** in the upper right: The heading is indicated in degrees, with 0º being North and 180º the South.
+
+**Controls**
+On this screen, the remote control buttons fulfill the following functions:
+- **Up**: Recalculates partial distance up (increases). If the precision of the distance is set to 1 decimal, adds 100 meters. If it is set to 2 decimal places, add 10 meters. Keeping the button pressed will quickly increase to do changes of great distances.
+- **Down**: Recall partial distance down (reduce). If the distance accuracy is set to 1 decimal, subtract 100 meters. If it is set to 2 decimal places, subtract 10 meters. Keeping the button pressed will quickly zoom out to make changes of great distances.
+- **Selection**: If pressed briefly, it switches to the next screen. If it stays pressed resets the partial distance to 0. This **DOES NOT** affect the total distance.
+
+### Speed
+
+On this screen you can see:
+- The speed at which you are currently moving.
+- The average speed since the last reset carried out from the configuration menu.
+**NOTE**: This average includes the time the motorcycle is not moving. This is especially useful to know your real average the regularity rallies.
+- The maximum speed reached since the last reset carried out from the settings menu setting. This value is useful to avoid being penalized in rallies where speed maximum is limited.
+**Controls**
+On this screen, the remote control buttons fulfill the following functions:
+- **Up**: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+
+
+`` ''
+- Bottom: If the QuickView functionality is enabled, displays the Odometer screen while holding down the button.
+- Selection: Short press switches to next screen. If it stays pressed it accesses the configuration screen.
+`` ''
+### Travel time
+
+On this screen you can see the time elapsed since the last reset performed since the settings menu, including the time the motorcycle has been stopped. This is especially useful for regularity rallies.
```

Comment:
```suggestion
In this screen you can see the time elapsed since the last total distance reset performed from the settings menu, including the time the motorcycle has been stopped. This is especially useful for regularity rallies.
```